pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -641,6 +650,8 @@ def ready(self):
 
     @property
     def status(self):
+        if self.CDN_used_as_installation_source:
```

Comment:
I am afraid of the side effects of this `if` that overrides all other states.

Reply:
By this we basically delegate the detailed source configuration to the Subscription spoke, which makes sure the subscription is attached properly enabling CDN use.

I guess there could still be some edge cases related to using CDN + custom additional repos, I'll make a note to investigate those as a followup.